core/comet/service.go

```diff
@@ -0,0 +1,55 @@
+package comet
```

Comment:
Should have package docs. I guess if this changes substantially we can create `comet/v2`, `comet/v3`, packages?